### PR TITLE
[WIP] Adding a time picker

### PR DESCRIPTION
**Pull request by inssein**
This is an initial implementation of TimePickerType.

Current Bugs:
1) Can't specify 12 hour format, partially because of https://github.com/symfony/symfony/issues/6587.
2) When specifying with_seconds, the layout is slightly broken

This is still a WIP, submitting a pull request for comments.

### CODE REVIEW

File: Date/MomentFormatConverter.php [hunk 1]
```diff
@@ -37,6 +37,31 @@ class MomentFormatConverter
         "d.m.Y, H:i:s"               => 'DD.MM.YYYY, HH:mm:ss',  // german format: 14.05.2014, 13:55:01
         "yyyy-MM-dd"                 => 'YYYY-MM-DD',            // 2014-05-14
         "Y-m-d"                      => 'YYYY-MM-DD',            // 2014-05-14
+
+        // 24 hour format
+        'G:i'                        => 'H:mm',
```
Comment (pulzarraider) [inline Date/MomentFormatConverter.php]: Seems that your are converting time from php `date` format. The [Intl formats](http://userguide.icu-project.org/formatparse/datetime#TOC-Date-Time-Format-Syntax) should be used instead.

**Comment by rewotec** [comment]
:+1:

**Comment by pulzarraider** [comment]
@inssein do you want to continue working on this PR?

**Comment by inssein** [comment]
@pulzarraider I'm sorry I don't have time, and it would take me a little back to get into this, as I haven't touched the project in over a year. If I remember this correctly, it was in an almost working state, but I am sure there has been many changes since that would require some merging.

**Comment by jorenvh1** [comment]
Anyone still working on this?

**Pull request closed without merging**